apps/web/src/hooks/useSwapCallback.ts

```diff
@@ -188,17 +184,14 @@ export function useSwapCallback(
             return response.hash
           })
           .catch((error: any) => {
-            // if the user rejected the tx, pass this along
-            if (error?.code === 4001) {
-              throw new Error('Transaction rejected.')
```

Comment:
should we keep this?

Reply:
Got it.

Reply:
@memoyil Let's do the same thing for farms. I think it's still popup the error when user reject